### ISSUE TITLE
Loot is too overpowered

### ISSUE THREAD

**Issue by gabrielpf3** [jtorleon-studios-team/bettervillages#121]
I'm a huge fan of this mod, but I always find the loot to be a bit too much for my tastes. This mod makes it WAY too easy to get full sets of iron armor as soon as you spawn in and even things that should be difficult to acquire and you shouldn't be able to get until late game, like Wither Skeleton Skulls. It'd be great if we could at least get a configuration option to nerf the loot.

**Comment by jtorleonstudios** [comment]
Yes you're right, we've already removed some of the armor in the local version, we'll continue to review the loot in the upcoming update.

**Comment by Jotave7** [comment]
> Yes you're right, we've already removed some of the armor in the local version, we'll continue to review the loot in the upcoming update.

The loots are really bad, the structures and villages are very good, but the loots don't make sense, honestly there should only be useless items in the chests and at most some iron things, no emeralds or things like that.

**Comment by jtl-elisa** [comment]
Done.
Will be available in the next update 3.2.2

**Comment by RHINE-LAB-Muelsyse** [comment]
> 是的，你是对的，我们已经删除了本地版本中的一些盔甲，我们将在即将到来的更新中继续审查战利品。

Oh no, I don't think directly removing specified loot is a very reasonable choice. A more reasonable option would be to provide a configuration file that allows players to choose between generating abundant loot or scarce loot. For example, I have installed a mod where stealing from villagers' chests lowers reputation and attracts attacks from iron golems. In this scenario, abundant loot is clearly a more suitable option, otherwise, the villagers' efforts to guard it would be less meaningful.
